test/path.jl

```diff
@@ -0,0 +1,5 @@
+@unix_only @test expanduser("~")[1] != ENV["HOME"]
+
+@unix_only @test isabspath("/") == true
+@test isabspath("~") == false
+@test isabspath(expanduser("~")) == true
```

Comment:
This should also be `@unix_only`, as `expanduser("~") == "~"` and `isabspath("~") == false` on Windows